public/app/whiteboards/list/list.html

```diff
@@ -10,8 +10,23 @@
   </div>
 </div>
 
+<div class="whiteboards-list-top">
```

Comment:
The feature requests for making whiteboards searchable is mostly to make the list more manageable to instructors. At this point, I would only show the search options to course admins.